### ISSUE TITLE
Error 500 for entire site

### ISSUE THREAD

**Issue by lareusmedia** [mzur/kirby-uniform#166]
I copied the repo into site/plugins/uniform and afterwards the entire site throws an error 500.

Here is the php error_log:

```
[17-Dec-2018 11:43:37 Europe/Berlin] PHP Warning:  require(/Users/christian/projekte/maxfeld/site/plugins/uniform/vendor/composer/../myclabs/deep-copy/src/DeepCopy/deep_copy.php): failed to open stream: No such file or directory in /Users/christian/projekte/maxfeld/site/plugins/uniform/vendor/composer/autoload_real.php on line 66
[17-Dec-2018 11:43:37 Europe/Berlin] PHP Fatal error:  require(): Failed opening required '/Users/christian/projekte/maxfeld/site/plugins/uniform/vendor/composer/../myclabs/deep-copy/src/DeepCopy/deep_copy.php' (include_path='.:/Applications/MAMP/bin/php/php7.2.8/lib/php') in /Users/christian/projekte/maxfeld/site/plugins/uniform/vendor/composer/autoload_real.php on line 66
```

Im using MAMP on Mac Mojave with PHP version 7.2.8

**Comment by mzur** [comment]
Should be fixed now with v4.0.0-beta2. Please try again.

**Comment by lareusmedia** [comment]
It is! Thank you!
